apps/cowswap-frontend/src/modules/permit/state/permittableTokensAtom.ts

```diff
@@ -15,6 +15,7 @@ export const permittableTokensAtom = atomWithStorage<PermittableTokens>('permitt
   [SupportedChainId.MAINNET]: {},
   [SupportedChainId.GOERLI]: {},
   [SupportedChainId.GNOSIS_CHAIN]: {},
+  [SupportedChainId.SEPOLIA]: {},
```

Comment:
Random thought: for this kind of thing where all networks need to be present and have a default value, could we create this map dynamically so for the next network added we don't need to change it again?

Tried a bit but couldn't get TS happy.

```ts
const m: PermittableTokens = Object.values(SupportedChainId).reduce((acc, v) => {
  acc[v] = {}
  return acc
}, {})
```

![image](https://github.com/cowprotocol/cowswap/assets/43217/02ae17bc-59ec-469c-b7f0-4cd556bdc527)

Reply:
https://github.com/cowprotocol/cow-sdk/pull/194